packages/editor/src/client/routes/authorList.tsx

```diff
@@ -52,7 +71,10 @@ export function AuthorList() {
 
   const authorListQueryVariables = {
     filter: filter || undefined,
-    first: 50
+    skip: page - 1,
+    sort: mapColumFieldToGraphQLField(sortField),
+    first: 50,
```

Comment:
Pagination doesn't work if you're always getting 50 articles. I think you have to use `limit` here.

Reply:
@roosnic1  Can you explain?

Reply:
Add more than 10 authors and test if pagination works with limit set to 10. If you have a limit of 10 you should only see 10 authors.